.flake8

```diff
@@ -1,5 +1,3 @@
 [flake8]
 import-order-style = google
 application-import-names = e2c, tests
-per-file-ignores =
-    tests/*: D104
```

Comment:
Why deleting this? You should not be required to have module docstring for `tests/`.